GraphConsoleAppV3/GraphRequests.cs

```diff
@@ -0,0 +1,1534 @@
+                }
+            }
+            return newUser;
+
+            #endregion
+        }
+
+        private static async Task UpdateNewUser(IUser newUser)
+        {
+            #region Update User properties
+
+            //*******************************************************************************************
+            // update the newly created user's Password, PasswordPolicies and City
+            //*********************************************************************************************
+            if (newUser.ObjectId != null)
+            {
+                // update User's info
+                newUser.City = "Seattle";
+                newUser.Country = "UK";
+                newUser.Mobile = "+4477889456789";
+                newUser.UserType = "Member";
+
+                try
+                {
+                    await newUser.UpdateAsync();
+                    Console.WriteLine("\nUser {0} was updated", newUser.DisplayName);
+                }
+                catch (Exception e)
+                {
+                    Program.WriteError("\nError Updating the user {0} {1}", e.Message,
+                        e.InnerException != null ? e.InnerException.Message : "");
+                }
+            }
+
+            #endregion
+        }
+
+        private static async Task ResetUserPassword(IUser newUser)
+        {
+            #region Reset password for user
+
+            //*******************************************************************************************
+            // update the newly created user's Password and PasswordPolicies
+            // requires Directory.AccessAsUser.All and that the current user is a user, helpdesk or company admin
+            //*********************************************************************************************
+            if (newUser.ObjectId != null)
+            {
+                // update User's password policy and reset password - forcing change password at next logon
+                PasswordProfile PasswordProfile = new PasswordProfile
+                {
+                    Password = "changeMe!",
+                    ForceChangePasswordNextLogin = true
+                };
+                newUser.PasswordProfile = PasswordProfile;
+                newUser.PasswordPolicies = "DisablePasswordExpiration, DisableStrongPassword";
+                try
+                {
+                    await newUser.UpdateAsync();
+                    Console.WriteLine("\nUser {0} password and policy was reset", newUser.DisplayName);
+                }
+                catch (Exception e)
+                {
+                    Program.WriteError("\nError Updating the user {0} {1}", e.Message,
+                        e.InnerException != null ? e.InnerException.Message : "");
+                }
+            }
+
+            #endregion
+        }
+
+        private static async Task OtherUserWriteOperations(IActiveDirectoryClient client, IUser newUser,
```

Comment:
Split off the assigning manager portion and renamed.

---
In reply to: [80543623](https://github.com/Azure-Samples/active-directory-dotnet-graphapi-console/pull/37#discussion_r80543623) [](ancestors = 80543623)